packages/Table/src/Table.tsx

```diff
@@ -82,6 +64,20 @@ const Table: FunctionComponent<ITableProps> = ({
     onSelect(checkedValue);
   };
 
+  const handleOnPageChange = (currentPage) => {
```

Comment:
I think we need to change how this component works to pass this event outside of the Table component so that the admin can fetch more data to display in the table.

some listing pages might have thousands of rows, we can't fetch them all and pass them to the component.

The component probably 
- should be given get a set of rows/data 
- on pagination change we call `handleOnPageChange` and fetch more data from the api and return it.